apps/client/src/app/core/data/lazy-data.service.ts

```diff
@@ -62,21 +61,30 @@ export class LazyDataService {
   public data: LazyData;
   public data$: ReplaySubject<LazyData> = new ReplaySubject<LazyData>();
 
+  public readonly fishingSpots$ = this.lazyDataProvider.getLazyData('fishingSpots');
```

Comment:
Besides the addition of this line, changes at L250 to accommodate the lazy file list structure change, this file is all changes made by prettier.